src/datasource/gateway.js

```diff
@@ -175,9 +175,9 @@ class PocketQueriesController {
           value: base64ToStr(kvObj.value),
         });
 
-        const mapEvents = (events) => events.map(
+        const mapEvents = (events) => events ? events.map(
```

Comment:
Previously when we were fetching out of the events, this was acting as a safe guard. Let's leave it in place if we ever need them. New transactions will not have this field but older ones will still have it.

Keeping this as a point of reference. Let me know wdyt